Source/Upstream: Yes, in v238-stable branch

From cb8a6583ba317db49057c1c710dd19e415f3eb5f Mon Sep 17 00:00:00 2001
From: Doug Christman <douglaschristman@gmail.com>
Date: Tue, 13 Mar 2018 16:41:07 +0800
Subject: [PATCH 17/51] zsh-completion: add calendar to systemd-analyze (#8438)

(cherry picked from commit ce21568dc34494f125bb70f6c51f80a5232b9020)
---
 shell-completion/zsh/_systemd-analyze | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/shell-completion/zsh/_systemd-analyze b/shell-completion/zsh/_systemd-analyze
index bccdb951f..56982e3d1 100644
--- a/shell-completion/zsh/_systemd-analyze
+++ b/shell-completion/zsh/_systemd-analyze
@@ -33,12 +33,13 @@ _systemd_analyze_command(){
         'plot:Output SVG graphic showing service initialization'
         'dot:Dump dependency graph (in dot(1) format)'
         'dump:Dump server status'
-        'unit-paths':List unit load paths'
+        'unit-paths:List unit load paths'
         'log-level:Get/set systemd log threshold'
         'log-target:Get/set systemd log target'
         'service-watchdogs:Get/set service watchdog status'
         'syscall-filter:List syscalls in seccomp filter'
         'verify:Check unit files for correctness'
+        'calendar:Validate repetitive calendar time events'
     )
 
     if (( CURRENT == 1 )); then
-- 
2.16.3

